src/Service/PaymentMethodsService.php

```diff
@@ -154,6 +144,31 @@ public function getPaymentMethods(SalesChannelContext $context, $orderId = ''):
         return $responseData;
     }
 
+    /**
+     * @param PaymentMethodsResponse $paymentMethodsResponse
+     * @return array
+     */
+    public function getPaymentMethodsArray(PaymentMethodsResponse $paymentMethodsResponse): array
+    {
+        $allPaymentMethods = [];
+        /** @var PaymentMethodsResponse $paymentMethods */
+        $paymentMethods = $paymentMethodsResponse->getPaymentMethods();
+        if (!empty($paymentMethods)) {
+            foreach ($paymentMethods as $paymentMethod) {
+                $allPaymentMethods['paymentMethods'][] =
+                    json_decode(json_encode($paymentMethod->jsonSerialize()), true);
```

Comment:
New version of the library has `toArray()` method on each object. Using that method would be more elegant than using encode/decode sequentially.